src/main/java/com/useful/ucars/uCarsListener.java

```diff
@@ -1275,20 +1293,13 @@ public void run() {
 				theNewLoc.add(0, 1.5d, 0);
 				Boolean calculated = false;
 				double y = 1.1;
-				if (block.getType().name().toLowerCase().contains("step")) {
+				if (block.getType().name().toLowerCase().contains("slab")) {
 					calculated = true;
 					y = 1.2;
 				}
-				if (carBlock.name().toLowerCase().contains("step")) { // In
-																		// a
-																		// step
-																		// block
-																		// and
-																		// trying
-																		// to
-																		// jump
+				if (carBlock.name().toLowerCase().contains("slab")) { // In a slab block
 					calculated = true;
-					y = 0.6;
+					y = 1.2;
```

Comment:
Does this definitely not work as 0.6? Would this being 1.2 not make the car float too high when driving over a road made of half slabs?

Reply:
I've just tried again and the lowest I can go is 1.1 (without going into 2 decimal digits)
Lower and you can not get of a slab-road.
You float as high as a normal block, yes.
It is actually similar to 0.6 float but here is the catch:
As soon as I get up to a block I drop.
This could have to do with some other jumping mechanics, I'll check.

Reply:
Hmm you should be able to be offset 0.6 when inside the slab, and then if you are trying to drive off of the road (into a full size block) detect that and then decide to climb - not be permanently too high for the slab

Reply:
Yes I'm probably gonna implement something like that.
Maybe the 0.6 offset wont even be necessary after that as the floating looks a bit odd...

Reply:
The 0.6 is to calculate what height it should be at I think, eg. block y coordinate+0.6 (which should be just above the top of the slab)

Reply:
Okay so I added that check in a slightly different way. Climbing and stuff works now BUT there is a Problem:

The slab elevator will make you float basically half a block over the slab.
What is that slab elevator even supposed to achieve?

Reply:
Nevermind that one, figured it out, will fix